### PR TITLE
[Composer] Reveal cloud_data_lineage_integration to GA.

### PR DESCRIPTION
**Pull request by ahidalgob**
Google Cloud Composer has promoted Data Lineage integration to GA.
Fixes https://github.com/hashicorp/terraform-provider-google/issues/19345

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
composer: promoted `config.0.workloads_config.0.cloud_data_lineage_integration` to GA in resource `google_composer_environment`
```

### CODE REVIEW

**Comment by ahidalgob** [comment]
```
make testacc TEST=./google/services/composer/ TESTARGS='-run=TestAccComposerEnvironment_ComposerV2'
TF_ACC=1 TF_SCHEMA_PANIC_ON_ERROR=1 go test ./google/services/composer/ -v -run=TestAccComposerEnvironment_ComposerV2 -timeout 240m -ldflags="-X=github.com/hashicorp/terraform-provider-google/version.ProviderVersion=acc"
=== RUN   TestAccComposerEnvironment_ComposerV2
=== PAUSE TestAccComposerEnvironment_ComposerV2
=== RUN   TestAccComposerEnvironment_ComposerV2HighResilience
=== PAUSE TestAccComposerEnvironment_ComposerV2HighResilience
=== CONT  TestAccComposerEnvironment_ComposerV2
=== CONT  TestAccComposerEnvironment_ComposerV2HighResilience
--- PASS: TestAccComposerEnvironment_ComposerV2 (1269.27s)
--- PASS: TestAccComposerEnvironment_ComposerV2HighResilience (1289.29s)
PASS
ok      github.com/hashicorp/terraform-provider-google/google/services/composer 1289.711s
```

File: mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown [hunk 1]
```diff
@@ -1037,7 +1037,7 @@ The following arguments are supported:
   major Cloud Composer or Apache Airflow versions (from `1.x.x` to `2.x.x`). To do so, create a new environment.
 
 * `cloud_data_lineage_integration` -
-  (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html),
+  (Optional,
   Cloud Composer environments in versions composer-2.1.2-airflow-*.*.* and newer)
```
Comment (NickElliot) [inline mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown:1041]:
```suggestion
  (Optional) Cloud Composer environments in versions composer-2.1.2-airflow-*.*.* and newer)
```

File: mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown [hunk 2]
```diff
@@ -1465,7 +1465,7 @@ The following arguments are supported:
   **Important**: In-place upgrade for Composer 3 is not yet supported.
 
 * `cloud_data_lineage_integration` -
-  (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html),
+  (Optional,
   Cloud Composer environments in versions composer-2.1.2-airflow-*.*.* and later)
```
Comment (NickElliot) [inline mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown:1469]:
```suggestion
  (Optional) Cloud Composer environments in versions composer-2.1.2-airflow-*.*.* and later)
```

**Review by NickElliot — CHANGES_REQUESTED** [review]
minor docs edits needed but otherwise seems good

File: mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown (hunk 2 above)
Comment (ahidalgob) [reply mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown]: same as previous

**Review by NickElliot — APPROVED** [review]
LGTM!

**Pull request merged**